packages/lit-html/src/lit-html.ts

```diff
@@ -1095,6 +1113,11 @@ export class EventPart extends AttributePart {
   }
 }
 
+// Apply polyfills if available
+if ((globalThis as any)['litHtmlPlatformSupport'] !== undefined) {
```

Comment:
Consider writing this as:

```ts
(globalThis as any)['litHtmlPlatformSupport']?.({NodePart, Template});
```

and it'll be compiled out depending on our tsconfig

Reply:
Done.